spec/aep-63/README.md

```diff
@@ -0,0 +1,131 @@
+
+While these endpoints will be made available to the user, we anticipate that most users will use the UI to manage their keys
+
+`POST /v1/users/api-keys` (create)
+`GET /v1/users/api-keys` (read/ list all keys)
+`GET /v1/users/api-keys/{id}` (read/ list specific API key details)
+`PATCH /v1/users/api-keys/{id}` (update)
+`DELETE /v1/users/api-keys/{id}` (delete)
+
+In addition there needs to be validation of the API keys at the middleware level so that it applies to any user specific endpoint
+
+Implementation/ status tracked in [Console issue #768](https://github.com/akash-network/console/issues/768)
+
+#### Certificate Management
+
+While there will be a documented API for this - it will likely be handled automagically under the hood for the managed wallet user. If it is handled for the user we may not implement the list and revoke endpoints
+
+`POST /v1/certificates` (create cert)
+`GET /v1/certificates` (list all certs)
+`DELETE /v1/certificates/{id}` (revoke cert)
+
+#### Deployment Creation
+
+`POST /v1/deployments` (create deployment based on `SDL/ YAML` and escrow `deposit`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Bid Selection
+
+`GET v1/bids/{dseq}` (get bids for a `dseq`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Lease Creation
+
+`POST /v1/leases` (create lease based on a payload that is an array of sdl, dseq, gseq, provider combinations)
+
+Note that Akash supports creating multiple leases from a single SDL - where each lease can be for a different service (with a different container image) and deployed to a different provider
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Listing Deployments
+
+`GET /v1/deployments` (list all deployments)
+`GET /v1/deployments/{dseq}` (list deployment detail for a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Deployment Closure
+
+`DELETE /v1/deployments/{dseq}` (close a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Funding Deployments
+
+For viewing or adding funds to the escrow of a specific deployment
+
+`GET /v1/deployments/deposit{dseq}` (retrieve escrow balance details)
+`POST /v1/deployments/deposit/{dseq}` (accepts `deposit` amount in the body)
+
+implementation tracked in [issue #989](https://github.com/akash-network/console/issues/989) and [issue #990](https://github.com/akash-network/console/issues/990)
+
+#### Funding Account
+
+This is for the customer to purchase more credits and fund their account - we will decide if we offer this or not based on customer requests but it will likely directly go to the Stripe Checkout API https://docs.stripe.com/api/checkout/sessions/object
+
+#### Retrieving Logs
+
+`GET/v1/logs/{dseq}`
```

Comment:
TBD - this usually requires `gseq` and `oseq`, but I don't think we want users to be confused with that so we might add it as options query params